package.json

```diff
@@ -47,7 +47,7 @@
     "slate-history": "^0.100.0",
     "slate-hyperscript": "^0.100.0",
     "slate-react": "^0.100.0",
-    "terraso-client-shared": "github:techmatters/terraso-client-shared#974ef95",
+    "terraso-client-shared": "github:techmatters/terraso-client-shared#f80b96103635ef5c856cca663fff5d64e2c85b82",
```

Comment:
issue: use short hash.

Reply:
This will be changed when I get the last version from the shared code after the other PRs are merged